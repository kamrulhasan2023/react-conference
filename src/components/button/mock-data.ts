import {
  IFillButton,
  IFillRedirectButton,
  IOutlineButton,
  IOutlineRedirectButton,
} from './interface';

export const buttonProps: IFillButton = {
  label: 'Button',
  colorScheme: 'bg-primary-text-white',
  hoverColorScheme: 'bg-success-text-white',
  size: 'lg',
  clickHandler: () => {},
  variant: 'fill',
};

export const redirectButtonProps: IFillRedirectButton = {
  href: 'https://www.google.com/',
  isOpenNewTab: true,
  label: 'Redirect Button',
  colorScheme: 'bg-primary-text-white',
  hoverColorScheme: 'bg-success-text-white',
  size: 'lg',
  variant: 'fill',
};

export const outlineButtonProps: IOutlineButton = {
  label: 'Outline Button',
  colorScheme: 'bg-transparent-text-primary',
  hoverColorScheme: 'bg-transparent-text-success',
  size: 'lg',
  clickHandler: () => {},
  variant: 'outline',
};

export const redirectOutlineButtonProps: IOutlineRedirectButton = {
  href: 'https://www.google.com/',
  isOpenNewTab: true,
  label: 'Outline Redirect Button',
  colorScheme: 'bg-transparent-text-primary',
  hoverColorScheme: 'bg-transparent-text-success',
  size: 'lg',
  variant: 'outline',
};
