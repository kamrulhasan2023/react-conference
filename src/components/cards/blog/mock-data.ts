import { IBlogCard } from './interface';

export const blogCardProps: IBlogCard = {
  href: 'https://www.lemonhive.com/',
  image: {
    src: 'https://res.cloudinary.com/dktw59lc2/image/upload/v1669267967/BrightonSEO/Carousel/image-2_yph8q6.jpg',
    alt: 'dummy image 2',
    lqip: 'https://res.cloudinary.com/dktw59lc2/image/upload/v1669267967/BrightonSEO/Carousel/image-2_yph8q6.jpg',
  },
  title: 'Headless Website Development',
  description:
    'Headless solutions built with Sanity.io, Strapi, Prismic, Contentful and others with front-end frameworks such as Next.js, Gatsby, & Nuxt.',
  tags: [
    { fontColor: { hex: 'black' }, bgColor: { hex: 'red' }, title: 'Reactjs' },
    { fontColor: { hex: 'black' }, bgColor: { hex: 'red' }, title: 'Nextjs' },
  ],
  isFeatured: false,
};
