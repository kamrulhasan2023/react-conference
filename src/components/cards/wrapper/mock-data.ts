import { ICardWrapper } from './interface';

export const cardWrapperProps: ICardWrapper = {
  cardBorderRadius: 'radius-none',
  cardShadow: 'shadow-one',
};
