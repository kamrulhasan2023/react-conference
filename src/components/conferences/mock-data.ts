import {IConferences} from "./interface";
export const conferencesProps:IConferences = {
    title : "Conferences",
    conferences: [
        {   
            id:1,
            title: "Freezing Edge 2023",
            description:"The edge isn't bleeding, it's freezing!",
            startDate: "02 September, 2023",
        },
        {
            id:2,
            title: "Freezing Edge 2023",
            description:"The edge isn't bleeding, it's freezing!",
            startDate: "02 September, 2023",
        },
        {
            id:3,
            title: "Freezing Edge 2023",
            description:"The edge isn't bleeding, it's freezing!",
          startDate: "02 September, 2023",
        },
        {
            id:4,
            title: "Freezing Edge 2023",
            description:"The edge isn't bleeding, it's freezing!",
            startDate: "02 September, 2023",
           
        },
        {
            id:5,
            title: "Freezing Edge 2023",
            description:"The edge isn't bleeding, it's freezing!",
            startDate: "02 September, 2023",
        }
    ]

}