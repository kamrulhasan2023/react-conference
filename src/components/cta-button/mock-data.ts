import { ICtaButton } from './interface';

export const ctaButtonProps: ICtaButton = {
  label: 'Cta button',
  href: '/',
  colorScheme: 'primary',
  isOpenNewTab: false,
};
