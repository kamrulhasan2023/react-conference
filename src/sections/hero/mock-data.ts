import { IHero } from './interface';

export const heroProps: IHero = {
  title: 'Lemon Hive, We Build Lightning Fast Websites',
  description:
    'We make secure, flexible and fast headless (Jamstack) websites and apps that improve Technical SEO, increase visitor numbers & conversion rates. We build with Sanity, BigCommerce, Prismic, Strapi, Contentful, WordPress, and other headless content management systems. We create front-ends with Vue, Svelte, Remix and React utilising Next.js, Gatsby, Nuxt, React Native, and others.',
  buttons: {
    buttonLeft: {
      label: 'Server Side Example',
      href: '/',
    },
    buttonRight: {
      label: 'Client Side Example',
      href: 'client-side',
    },
  },
};
