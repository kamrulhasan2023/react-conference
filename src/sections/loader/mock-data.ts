import { ILoaderSection } from './interface';

export const loaderSectionProps: ILoaderSection = {
  level: 'section',
};
