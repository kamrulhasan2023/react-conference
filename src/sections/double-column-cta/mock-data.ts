import { IDoubleColumnCtaSection } from './interface';
import { sectionDescriptionProps } from '@components/section-heading/mock-data';

export const doubleColumnCtaSectionProps: IDoubleColumnCtaSection = {
  sectionHeading: {
    title:
      'Weâ€™d love to help start your headless journey or  to help optimize your existing solution.',
  },
  rightContent: sectionDescriptionProps,
  ctaButton: {
    label: 'Book a project discovery call',
    href: 'https://example.com/',
    isOpenNewTab: true,
  },
};
