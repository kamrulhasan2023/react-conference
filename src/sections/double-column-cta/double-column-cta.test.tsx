import { render } from '@testing-library/react';
import DoubleColumnCtaSection from '.';
import { doubleColumnCtaSectionProps } from './mock-data';

it('renders double column cta section correctly', () => {
  const { getAllByRole } = render(
    <DoubleColumnCtaSection {...doubleColumnCtaSectionProps} />
  );
  const element = getAllByRole('presentation')[0];

  expect(element).toBeInTheDocument();
  expect(element).toMatchSnapshot();
});
