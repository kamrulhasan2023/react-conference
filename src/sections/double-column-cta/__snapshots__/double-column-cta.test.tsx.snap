// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders double column cta section correctly 1`] = `
<section
  class="section-padding-primary bg-primary-thin"
  role="presentation"
>
  <div
    class="container"
    role="presentation"
  >
    <div
      class="flex flex-col items-center justify-between gap-6 md:flex-row"
    >
      <div
        class="w-full md:w-[48%] lg:w-[45%] xl:w-[42%] 2xl:w-[40%]"
      >
        <div
          class="section-heading black"
          role="presentation"
        >
          <h2
            class="text-2xl font-medium  md:text-4xl md:font-bold title"
          >
            Weâ€™d love to help start your headless journey or  to help optimize your existing solution.
          </h2>
        </div>
      </div>
      <div
        class="w-full md:w-1/2"
      >
        <article
          class="colored-bg normal-block-font-size-normal"
        >
          <div
            class="space-y-4 main-wrapper"
          >
            <p
              class="normal"
              id="0ffdc2e0f0b8"
            >
              <a
                class="underline text-secondary"
                href="https://www.google.com"
                target="_blank"
              >
                Headless website
              </a>
               for an award winning digital marketing agency.
            </p>
            <br />
            <p
              class="normal"
              id="c0b3d76ed804"
            >
              <strong>
                Lorem ipsum
              </strong>
               dolor 
              <em>
                sit amet consectetur
              </em>
              . Vitae cursus ultricies interdu ridiculus. Nullam nuwel craegestas risus 
              <span
                style="text-decoration: underline;"
              >
                diam lorem nascetur
              </span>
              . Lorem ipsum dolor sit amet consectetur. Vitae cursus ultricies interdu ridiculus. Nullam nuwel craegestas risus 
              <strong>
                diam lorem nascetu ullam nuwel
              </strong>
            </p>
          </div>
        </article>
        <div
          class="mt-8 md:mt-9 xl:mt-11 2xl:mt-12"
        >
          <a
            class="relative z-10 overflow-hidden inline-flex items-center justify-center underline transition-colors cta-button primary"
            href="https://example.com/"
            target="_blank"
          >
            Book a project discovery call
          </a>
        </div>
      </div>
    </div>
  </div>
</section>
`;
